### DISCUSSION TITLE
Using the same Vercel domain with application running on cloud server

### DISCUSSION THREAD

**Discussion by jaynguyen2309** [vercel/vercel#8617] [Help]
Currently, we are deploying our landing page and the web application in same cloud server. They are configure with the same domain:

- Landing page: `landingpage.com` (Next.js)
- Application: `landingpage.com/platform` (React.js)

Currently, I'm using nginx to configure to 2 application point to the same domain. The `location /` will point to the landingpage port, and the `location /platform` will point to the application port.

I'm thinking about switching the deployment of the landing page to Vercel. Is there any chance that we can do the same as my current configuration with the same domain, but on Vercel for the landing page, and on the cloud server for the web application?

**Comment by amyegan** [comment]
Hi @duy23092000! You could use [rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites) to accomplish this behavior. See "[How can I serve multiple projects under a single domain?](https://vercel.com/guides/how-can-i-serve-multiple-projects-under-a-single-domain#deploying-rewrite-rules)" for an overview.

If you're considering incremental adoption, you should look through this thread where some options, pros, and cons are discussed: https://github.com/vercel/vercel/discussions/5769#discussioncomment-322453

**Reply by jaynguyen2309** [reply]
Hi @amyegan . Thanks for your reply. According to the [rewrites](https://nextjs.org/docs/api-reference/next.config.js/rewrites), the two project will be configured with 2 separate domains, while I want it to be the same, but different projects. Will that be possible?

**Reply by amyegan** [reply]
They would technically be under different domains, but would appear to be the same domain for users if you use rewrites. You could use a subdomain to get the same root domain for both (i.e. `landingpage.com` and `platform.landingpage.com`). But you would need to use rewrites to make `platform.landingpage.com` look like it's coming from `landingpage.com/platform`
